src/asRendition.tsx

```diff
@@ -92,11 +92,22 @@ export const withStyledSystem = (child: React.ComponentType) => {
 
 export const withTooltip = (Base: React.ComponentType) => {
 	return React.forwardRef(({ ...props }: any, ref: any) => {
-		if (props.tooltip) {
-			props = tooltip.bindProps(props);
+		if (!props.tooltip) {
+			return <Base {...props} ref={ref} />;
 		}
+		const tooltipProps = tooltip.bindProps(props);
 		delete props.tooltip;
-		return <Base {...props} ref={ref} />;
+		return props.disabled ? (
+			<span
+				onClick={tooltipProps.onClick}
```

Comment:
@JSReds will this allow users to trigger the `onClick` even on disabled buttons?

Reply:
As I understood it will show the tooltip if you click on it (for mobile devices, for example), but I might have misunderstood

Reply:
@thgreasi @sradevski I confirm that this is for mobile devices since there is no mouseEnter event